api/room/schema_query.py

```diff
@@ -297,7 +299,15 @@ def resolve_all_remote_rooms(self, info, return_all=None):
             page_token = calendar_list.get('nextPageToken')
             if not page_token:
                 break
-        return AllRemoteRooms(rooms=remote_rooms)
+        match = r'\bTest|\bDummy|\btest|\bdummy'
+        if os.getenv("APP_SETTINGS") == "production":
```

Comment:
@mwinel I think it is better to check if the environment is `not production` and use an `else`. Our environments are not limited to `production` and `development`.